datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GraphQLEngine.java

```diff
@@ -152,7 +155,7 @@ public GraphQLEngine build() {
         }
     }
 
-    private DataLoaderRegistry createDataLoaderRegistry(final Map<String, Supplier<DataLoader<?, ?>>> dataLoaderSuppliers) {
+    public DataLoaderRegistry createDataLoaderRegistry(final Map<String, Supplier<DataLoader<?, ?>>> dataLoaderSuppliers) {
```

Comment:
Would it make sense to expose a method called `getDataLoaderRegistry` that returns a cached version of a registry that was already created? 

That way, the caller does not need to explicitly pass in the `dataLoaderSuppliers`

Reply:
I agree! 👍